### ISSUE TITLE
Urgent! Stopped working due to new Signature Algorithm after June 30, 2021 on Tuya IoT Platform

### ISSUE THREAD

**Issue by BrOrlandi** [codetheweb/tuyapi#492]
**Describe the bug**
The wizard stopped working due to a new signature verification algorithm for projects after June 30, 2021

**To Reproduce**
1. Try to setup new projects created after June 30, 2021 running 

**Expected behavior**
Should setup correctly

**Debug Output**

```
[
  Error: sign invalid
      at OpenAPI._client.got_1.default.extend.hooks.afterResponse (/Users/brunoorlandi/.nvm/versions/node/v14.15.4/lib/node_modules/@tuyapi/cli/node_modules/@tuyapi/openapi/dist/api.js:74:31)
      at EventEmitter.<anonymous> (/Users/brunoorlandi/.nvm/versions/node/v14.15.4/lib/node_modules/@tuyapi/cli/node_modules/got/dist/source/as-promise.js:87:38)
      at processTicksAndRejections (internal/process/task_queues.js:93:5),
  Error: sign invalid
      at OpenAPI._client.got_1.default.extend.hooks.afterResponse (/Users/brunoorlandi/.nvm/versions/node/v14.15.4/lib/node_modules/@tuyapi/cli/node_modules/@tuyapi/openapi/dist/api.js:74:31)
      at EventEmitter.<anonymous> (/Users/brunoorlandi/.nvm/versions/node/v14.15.4/lib/node_modules/@tuyapi/cli/node_modules/got/dist/source/as-promise.js:87:38)
      at processTicksAndRejections (internal/process/task_queues.js:93:5),
  Error: sign invalid
      at OpenAPI._client.got_1.default.extend.hooks.afterResponse (/Users/brunoorlandi/.nvm/versions/node/v14.15.4/lib/node_modules/@tuyapi/cli/node_modules/@tuyapi/openapi/dist/api.js:74:31)
      at EventEmitter.<anonymous> (/Users/brunoorlandi/.nvm/versions/node/v14.15.4/lib/node_modules/@tuyapi/cli/node_modules/got/dist/source/as-promise.js:87:38)
      at processTicksAndRejections (internal/process/task_queues.js:93:5),
  Error: sign invalid
      at OpenAPI._client.got_1.default.extend.hooks.afterResponse (/Users/brunoorlandi/.nvm/versions/node/v14.15.4/lib/node_modules/@tuyapi/cli/node_modules/@tuyapi/openapi/dist/api.js:74:31)
      at EventEmitter.<anonymous> (/Users/brunoorlandi/.nvm/versions/node/v14.15.4/lib/node_modules/@tuyapi/cli/node_modules/got/dist/source/as-promise.js:87:38)
      at processTicksAndRejections (internal/process/task_queues.js:93:5)
]
There was an issue fetching that device. Make sure your account is linked and the ID is correct.
```

**Screenshots**
Warning on Tuya IoT platform:
![image](https://user-images.githubusercontent.com/1594658/124002069-b2f57d80-d9ab-11eb-8aab-0b951efc3fc1.png)

Reference:
https://developer.tuya.com/en/docs/iot/singnature?id=Ka43a5mtx1gsc#title-1-Token%20Management%20API%20Signature%20Algorithm

**Desktop (please complete the following information):**
 - OS: macOS
 - OS Version 11.4
 - Node Version v14.15.4

**Additional context**
I guess any projects created after June 30, 2021 will not be able to sign until it gets fixed on the cli.

**Comment by codetheweb** [comment]
I probably won't get to this for a few days but happy to accept PRs. :)

**Comment by generalk25** [comment]
same issue.

**Comment by csthomas2** [comment]
Same issue. Should have done this a week ago when I originally planned!

**Comment by codetheweb** [comment]
Fixed in @tuyapi/cli 1.17.0.

Note: https://github.com/TuyaAPI/cli/issues/130
